### PR TITLE
Remove reference to !secret in Todoist documentation

### PR DESCRIPTION
**Pull request by mamessner**
The paragraph below references secrets, but there isn't a secret in the config.

We could also just remove the paragraph below since this introduces some inconsistency (saying "YOUR_API_TOKEN" above and "!secret todoist_api_token" here).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @mamessner, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21473).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/todoist.markdown [hunk 1]
```diff
@@ -64,7 +64,7 @@ Creating custom projects is super-easy and quite powerful. All you need to run t
 # Example configuration.yaml entry
 calendar:
   - platform: todoist
-    token: YOUR_API_TOKEN
+    token: !secret todoist_api_token
```
Comment (frenck) [inline source/_integrations/todoist.markdown]: We should remove the secret note below instead and keep the constant here.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/todoist.markdown (hunk 1 above)
Comment (mamessner) [reply source/_integrations/todoist.markdown]: Done

**Review by frenck — APPROVED** [review]
Thanks, @mamessner 👍

**Pull request merged**